app/ide-desktop/eslint.config.js

```diff
@@ -461,11 +449,6 @@ export default [
                     selector: ':not(TSModuleDeclaration)[declare=true]',
                     message: 'No ambient declarations',
                 },
-                {
```

Comment:
This rule makes no sense in case if we re-import default statements from barrel files